GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeView.swift

```diff
@@ -0,0 +1,165 @@
+import UIKit
+
+extension CallVisualizer {
+    final class VisitorCodeView: BaseView {
+        struct Props: Equatable {
+            enum ViewType: Equatable {
+                case alert(closeButtonTap: Cmd)
+                case embedded
+            }
+
+            let viewType: ViewType
+            let style: AlertStyle
```

Comment:
Is it possible to pass style via Props (here and also for other views and view controller(s) as well)?

Reply:
fixed [493843d](https://github.com/salemove/ios-sdk-widgets/pull/448/commits/493843db355a590b59baac91fe37885aac075cfc)